vivisect/__init__.py

```diff
@@ -606,6 +611,9 @@ def analyze(self):
         Call this to ask any available analysis modules
         to do their thing...
         """
+        if self.getMeta('GUID') == None:
```

Comment:
i'm concerned about lingering "multiple GUID" events.  every time you open a workspace and load it, you'll have two events, and the first one will constantly change.  it seems ugly and unnecessary?      from a "hackish" level, "diff foo.viv bar.viv" will differ, even if they have exactly the same events, except the initial one.  unless we want GUID to be a top-level thing, which gets put into the file header, and not stored as an event.  thoughts?